content/applications/hr/referrals/share_jobs.rst

```diff
@@ -0,0 +1,165 @@
+
+The specific title of the job position populates the *Job Position* placeholder in the email body.
+The `See Job Offer` text in the email body is an individualized tracking link to the specific job
+position listed on the website.
+
+When the prospective employee receives the email, the link sends them to the job position page,
+where they can apply for the position, and the person who referred them is tracked in the
+*Referrals* application.
+
+If desired, add any text or closing salutation to the email body. When all edits have been made,
+click :guilabel:`Send Mail` to send the email, or click :guilabel:`Cancel` to close the pop-up
+window.
+
+.. image:: share-jobs/email.png
+   :align: center
+   :alt: Referral email pop-up window with the email message inside it.
+
+Share a job
+===========
+
+Other than sending an email, job positions can be shared, via social media platforms, and by
+tracking links to the job position. At the bottom of each job position card are four icons, and
+corresponding tracking links, that can be used to share the job position, keeping track of
+applicants in the *Referrals* application.
+
+.. image:: share-jobs/share.png
+   :align: center
+   :alt: The various sharing icons that appear for each job.
+
+Link
+----
+
+To share the job position with a customized tracking link, click the :guilabel:`Share Now` button
+with the :icon:`fa-chain` :guilabel:`(link)` icon above it. A :guilabel:`Link to Share` pop-up
+window appears with the tracking link. Click :guilabel:`Copy` to copy the link. After the link is
+copied, click the :guilabel:`Close` button to close the pop-up window. Next, share the link with
+the prospective employee.
+
+Facebook
+--------
+
+To share the job position using Facebook, click the :guilabel:`Share Now` button with the
+:icon:`fa-facebook` :guilabel:`(Facebook)` icon above it.
+
+If the user is already logged into Facebook, when the the :guilabel:`Share Now` button is clicked, a
+:guilabel:`Share on Facebook` page loads in a new tab, with the link populated in the main body of
+the new post in a pop-up window. If the user is *not* already logged in, a log-in screen loads,
+instead, prompting the user to log-in to Facebook first.
+
+Type in any additional information to add to the post, then share the job position using the
+available options in Facebook.
+
+X (formerly Twitter)
+--------------------
+
+A job position can also be shared on X. Click the :guilabel:`Share Now` button with the
+:guilabel:`(X)` icon above it.
+
+If the user is already signed-in to X, when the :guilabel:`Share Now` button is clicked, an X page
+loads in a new tab with a pre-populated message ready to post, in a draft pop-up window. If the user
+is *not* already signed-in, a sign-in screen loads instead, prompting the user to first sign-in to
+X.
+
+The default message is:
+
+:guilabel:`Amazing job offer for (Job Position)! Check it live: (link to Job Position)`
```

Comment:
Nitpick, if this is user inputed/editable text it should have literal formatting :)
```suggestion
`Amazing job offer for (Job Position)! Check it live: (link to Job Position)`
```